# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/wr/wrench/reftests/aa/reftest.list
# Commit: 8d13c0ab66f0
# Full Hash: 8d13c0ab66f0ac137b1461d3451ecb9361ec2d5e
# Author: Jamie Nicol <jnicol@mozilla.com>
# Date: 2023-03-22 21:15:54
# Regressor Bug: 1823411
# File Overlap Count: 1
# Description:
#   Bug 1823411 - Give all webrender's shader varyings an explicit precision. r=gfx-reviewers,lsalzman
#   
#   Mali profiling tools have shown we are frequently fragment bound, in
#   particular due to varying interpolation. To help mitigate this, we
#   should use mediump where possible. Currently most of our varyings are
# ==============================================================================

diff -r 3919fbd972a6 -r 8d13c0ab66f0 gfx/wr/wrench/reftests/aa/reftest.list
--- a/gfx/wr/wrench/reftests/aa/reftest.list	Tue Mar 21 19:04:22 2023 +0000
+++ b/gfx/wr/wrench/reftests/aa/reftest.list	Tue Mar 21 19:05:19 2023 +0000
@@ -1,3 +1,3 @@
 skip_on(android) fuzzy(1,1) fuzzy-if(platform(swgl),4,27) == rounded-rects.yaml rounded-rects-ref.png  # Too wide for Android
 == aa-dist-bug.yaml aa-dist-bug-ref.yaml
-== fractional-radii.yaml fractional-radii-ref.yaml
+fuzzy-if(env(android,device),6,792) == fractional-radii.yaml fractional-radii-ref.yaml